test/calculation/SanityCheckTest.cpp

```diff
@@ -0,0 +1,179 @@
+#include "gtest/gtest.h"
+
+#include <array>
+#include <utility>
+#include <tuple>
+#include "polyhedralGravity/calculation/SanityCheck.h"
+#include "polyhedralGravity/model/Polyhedron.h"
+#include "polyhedralGravity/input/TetgenAdapter.h"
+
+
+/**
+ * Contains Tests if the sanity checks works
```

Comment:
two more sentences might be nice. In which way are the cubes "correct" or "wrong"?